src/utils/index.js

```diff
@@ -79,3 +82,27 @@ export const parseJwt = (token) => {
         ? JSON.parse(Buffer.from(token.split('.')[1], 'base64').toString())
         : ''
 }
+
+/**
+ * Lấy dữ liệu từ firebase
+ * @param {*} collectionRef dtb firebase
+ * @param {*} conditions điều kiện query
+ * @param {*} setDataCallback trả về usestate dữ liệu
+ * @returns useState
+ */
+
+export const useFirestoreQuery = (collectionRef, conditions, setDataCallback) => {
```

Comment:
This is a hook not an util function. You should move it to `src/hooks` and create new file for this hooks.